src/components/TileGallery/TileGalleryItem.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  /** Card title */
+  title: PropTypes.string.isRequired,
+  /** Card description */
+  description: PropTypes.string,
+  /** More info link */
+  moreInfoLink: PropTypes.string,
+  /** More info text */
+  descriptionMoreInfo: PropTypes.string,
+  /** Bottom left  icon  */
+  icon: PropTypes.node,
```

Comment:
This should mirror [Carbon's icon proptype definition ](https://github.com/carbon-design-system/carbon/blob/v10.7.3/packages/react/src/components/Icon/Icon.js#L189-L194)

```jsx
icon: PropTypes.shape({
    width: PropTypes.string,
    height: PropTypes.string,
    viewBox: PropTypes.string.isRequired,
    svgData: PropTypes.object.isRequired,
  }),
```